xmtp_api_d14n/src/endpoints/v3/mls/subscribe_welcome_messages.rs

```diff
@@ -0,0 +1,67 @@
+use derive_builder::Builder;
+use prost::Message;
+use prost::bytes::Bytes;
+use std::borrow::Cow;
+use xmtp_proto::api::{BodyError, Endpoint};
+use xmtp_proto::mls_v1::SubscribeWelcomeMessagesRequest;
+use xmtp_proto::mls_v1::subscribe_welcome_messages_request::Filter as SubscribeFilter;
+
+/// Query a single thing
+#[derive(Debug, Builder, Default, Clone)]
+#[builder(build_fn(error = "BodyError"))]
+pub struct SubscribeWelcomeMessages {
+    #[builder(setter(each(name = "filter", into)))]
+    filters: Vec<SubscribeFilter>,
+}
+
+impl SubscribeWelcomeMessages {
+    pub fn builder() -> SubscribeWelcomeMessagesBuilder {
+        Default::default()
+    }
+}
+
+impl Endpoint for SubscribeWelcomeMessages {
+    type Output = xmtp_proto::mls_v1::GroupMessage;
+
+    fn http_endpoint(&self) -> Cow<'static, str> {
```

Comment:
since the removal of api_http, we are no longer using http_endpoint, but saving cleanup for a new pr